BibBuddy/app/src/main/java/de/bibbuddy/AuthorRecyclerViewAdapter.java

```diff
@@ -96,6 +103,24 @@ public int getItemCount() {
     return authorItemList.size();
   }
 
+  /**
+   * This method fetches the number of items selected in the recyclerView.
+   *
+   * @return returns the selected recyclerView items.
+   */
+  public List<AuthorItem> getSelectedAuthorItems() {
+    List<AuthorItem> selectedItems = new ArrayList<>();
+    if (parent != null) {
+      int itemNumber = parent.getChildCount();
+      for (int i = 0; i < itemNumber; i++) {
+        if (parent.getChildAt(i).isSelected()) {
+          selectedItems.add(authorItemList.get(i));
+        }
+      }
+    }
+    return selectedItems;
```

Comment:
Please add some newlines for better readability. Also you don't need a local variable for itemNumber.
```suggestion
    List<AuthorItem> selectedItems = new ArrayList<>();
    
    if (parent != null) {
      for (int i = 0; i < parent.getChildCount(); i++) {
        if (parent.getChildAt(i).isSelected()) {
          selectedItems.add(authorItemList.get(i));
        }
      }
    }
    
    return selectedItems;
```

Reply:
Thanks for the suggestion, done!